### PR TITLE
Showcase: Validate timestamps.

### PR DESCRIPTION
**Pull request by ChrisAntaki**

### CODE REVIEW

File: src/runtime/entitlements-manager.js [hunk 1]
```diff
@@ -697,25 +692,41 @@ export class EntitlementsManager {
...
+
+            const attributeNames = Object.keys(attributes);
+            attributeNames.forEach((attributeName) => {
+              const name = `${category}_${attributeName}`;
+              const timestamp = attributes[attributeName].timestamp;
+
+              // Validate timestamp.
+              const timestampIsTooFarInTheFuture =
+                timestamp > (Date.now() / 1000) * 2;
+              if (!timestamp || timestampIsTooFarInTheFuture) {
```
Comment (ed-bird) [inline src/runtime/entitlements-manager.js:709]: Can we add another check if the timestamp is actually an integer instead of something else like "Friday, 7 May 2021 18:16:51"?

Could be a different PR if that's better.

Thanks!

**Review by ed-bird — APPROVED** [review]
Thanks @ChrisAntaki !

File: src/runtime/entitlements-manager.js (hunk 1 above)
Comment (ChrisAntaki) [reply src/runtime/entitlements-manager.js:709]: Good call, updated!
